easybuild/easyconfigs/r/r-inla/r-inla-21.01.18-foss-2020b.eb

```diff
@@ -0,0 +1,77 @@
+easyblock = 'MakeCp'
+
+name = 'r-inla'
+version = '21.01.18'
+
+homepage = 'https://www.r-inla.org/'
+description = """R-INLA is a package in R that do approximate Bayesian inference for Latent Gaussian Models."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://hrue.r-inla-download.org/']
+sources = [{
+    'download_filename': 'build64.tgz',
```

Comment:
Based on https://www.r-inla.org/download-install and https://inla.r-inla-download.org/R/stable/src/contrib/, we can avoid using a pre-built binary (and unversioned...) tarball?

Reply:
okay, me and @Darkless012 just tried this approach...firstly it really does look like a proper R-package approach, it installs and all and can be imported in R..
but when i tried to run test script from R-INLA man page, it failed and we noticed that some libraries are being taken (linked) from incorrect paths (usr/lib) which implies that they havent been created by easybuild...so i tried to remove some prebuilt binary files from inside the downloaded tarball that you sent me and install it afterwards...and indeed, these binaries do not appear in the final `bin` folder as they did before i tried to remove them from downloaded 'source code' so i guess it has to mean that the main thing R-INLA's 'installation process' does, is that it copies stuff from one folder to another, not actually compiling anything...i hope it's clear enough and i think we could have a further chat about this...